src/lib/operators/delete.cpp

```diff
@@ -98,6 +98,7 @@ void Delete::_on_commit_records(const CommitID cid) {
       auto referenced_chunk = referenced_table->get_chunk(row_id.chunk_id);
 
       referenced_chunk->get_scoped_mvcc_data_lock()->end_cids[row_id.chunk_offset] = cid;
+      referenced_chunk->increase_invalid_row_count(1);
```

Comment:
Would it be sufficient to do this once at the end?

Reply:
Within each iteration `referenced_chunk` changes. Therefore, to perform a "batch-increase" of a chunk's `invalid_row_count`, a container would be required to store the accumulated increase-count per chunk (thinking of `unordered_set`). Then, a single function call of `increase_invalid_row_count(..)` per chunk would be sufficient.

Might be more performant. Do you prefer this approach?

Reply:
we discussed this.